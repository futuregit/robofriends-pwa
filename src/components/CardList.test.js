import { shallow } from 'enzyme';
import toJson from "enzyme-to-json";
import React from 'react';
import CardList from './CardList';

it('expect to render Card component', () => {
    const mockRobots = [
        {
            id: 1,
            name: 'Fred Snow',
            username: 'FredFred',
            email: 'Fred@email.com'
        },

        {
            id: 2,
            name: 'Wilma Snow',
            username: 'Wilma',
            email: 'Wilma@email.com'
        }
    ]
    const wrapper = shallow(<CardList robots={mockRobots}/>);
    expect(toJson(wrapper)).toMatchSnapshot();
})