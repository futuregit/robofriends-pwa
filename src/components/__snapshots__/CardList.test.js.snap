// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card component 1`] = `
<div>
  <Card
    email="Fred@email.com"
    id={1}
    key="0"
    name="Fred Snow"
  />
  <Card
    email="Wilma@email.com"
    id={2}
    key="1"
    name="Wilma Snow"
  />
</div>
`;
